reference/7.0/Microsoft.PowerShell.Core/About/about_Redirection.md

```diff
@@ -202,17 +202,17 @@ Inquire
 The redirection operators that do not append data (`>` and `n>`) overwrite the
 current contents of the specified file without warning.
 
-However, if the file is a read-only, hidden, or system file, the
-redirection **fails**. The append redirection operators (`>>` and `n>>`) do not write
-to a read-only file, but they append content to a system or hidden file.
+However, if the file is a read-only, hidden, or system file, the redirection
+**fails**. The append redirection operators (`>>` and `n>>`) do not write to a
+read-only file, but they append content to a system or hidden file.
 
 To force the redirection of content to a read-only, hidden, or system file,
 use the `Out-File` cmdlet with its `Force` parameter.
 
-When you are writing to files, the redirection operators use Unicode encoding.
-If the file has a different encoding, the output might not be formatted
-correctly. To redirect content to non-Unicode files, use the `Out-File` cmdlet
-with its `Encoding` parameter.
+When you are writing to files, the redirection operators use UTF8 encoding
+(without a BOM). If the file has a different encoding, the output might not be
```

Comment:
Should BOM get spelled out?  I'm not sure everyone will understand this pseudo-comment.

Reply:
stet